lang/perl-mail-spamassassin/Makefile

```diff
@@ -28,10 +28,10 @@ define Package/spamassassin
   CATEGORY:=Mail
   TITLE:=SpamAssassin
   URL:=https://spamassassin.apache.org/
-  DEPENDS:=+perl +perlbase-autoloader +perlbase-config +perlbase-data +perlbase-digest \
-           +perlbase-encode +perlbase-essential +perlbase-file +perlbase-getopt \
-           +perlbase-hash +perlbase-mime +perlbase-net +perlbase-socket \
-           +perl-dbi +perl-html-parser +perl-net-dns +perl-netaddr-ip
+  DEPENDS:=perl +perlbase-autoloader +perlbase-config +perlbase-data +perlbase-digest \
+          +perlbase-encode +perlbase-essential +perlbase-file +perlbase-getopt \
+          +perlbase-hash +perlbase-mime +perlbase-net +perlbase-socket \
+          +perl-dbi +perl-html-parser +perl-net-dns +perl-netaddr-ip
```

Comment:
Please avoid whitespace only changes in the future...

Reply:
He removed the `+` indicating a _selects_ instead of a _depend on_ being generated in Kconfig. The other lines have been changed to fix the alignment, I suppose.

Reply:
Correct.